instaloader/structures.py

```diff
@@ -936,6 +936,11 @@ def followers(self) -> int:
     def followees(self) -> int:
         return self._metadata('edge_follow', 'count')
 
+    @property
+    def bio_links(self) -> List[str]:
+        """list of external links given in the bio"""
```

Comment:
Please also add a `versionadded` directive here, indicating it is new in version 4.12.